src/NCI.OCPL.Api.Glossary/Services/ESTermsQueryService.cs

```diff
@@ -200,16 +200,15 @@ public async Task<GlossaryTermResults> GetAll(string dictionary, AudienceType au
             }
             catch (Exception ex)
             {
-                String msg = String.Format("Could not get dictionary '{0}', audience '{1}', language '{2}', size '{3}', from '{4}'.", dictionary, audience, language, size, from);
+                String msg = $"Could not get dictionary '{dictionary}', audience '{audience}', language '{language}', size '{size}', from '{from}'.";
                 _logger.LogError("Error Fetching All from index: " + this._apiOptions.AliasName);
```

Comment:
Can you change this to the formatter please

Reply:
This should be for the line, `_logger.LogError("Error Fetching All from index: " + this._apiOptions.AliasName);`. GH review tool is adding this someplace odd...